login/core/validations/user_validation/DbValidation.js

```diff
@@ -1,14 +1,14 @@
 const Sequelize = require('sequelize');
 
 const SequelizeOperators = Sequelize.Op;
-const UserModel = require('../../db/models/UserModel.js');
-const createUserAccount = require('../../db/CreateUserAccount.js');
+const UserModel = require('../../../db/models/user_models/UserModel.js');
+const createUserAccount = require('../../../db/user_db/CreateUserAccount.js');
 
-function UserExist(registrationRequest, response) {
+function CopyCheck(registrationRequest, response) {
   UserModel.User.findOne({
     raw: true,
     where: {
-      [SequelizeOperators.or]: [{ login: registrationRequest.login }, { email: registrationRequest.email }],
+      [SequelizeOperators.or]: [{ login: registrationRequest.login }, { mail: registrationRequest.mail }],
```

Comment:
Shouldn't this be email? I saw email in some other file and honestly, email tells me more and is more common to describe an electronic mail address, which it basically is.